demo/x11_rawfb/nuklear_rawfb.h

```diff
@@ -130,19 +154,22 @@ nk_rawfb_line_horizontal(const struct rawfb_context *rawfb,
 }
 
 static void
-nk_rawfb_imagesetpixel(const struct rawfb_image *img,
+nk_image_setpixel(const struct rawfb_image *img,
```

Comment:
Could you please add some sane (short and unused by other backends) prefix to the function name (after `nk_`) and change also all other functions accordingly? I know you didn't introduce this mistake (it's our fault, that we approved previous pulls which introduced it), but when we're already by renaming...

The reason is to separate backends (but allow their parallel use - e.g. for runtime switching of backends) from Nuklear as backends are officially maintained by the community and not the Nuklear project (therefore backends are also not versioned and can change without a notice any time).

Second reason is to avoid confusion if Nuklear wanted to introduce similarly called function in the future.

Reply:
Makes sense. Left the bulk of the `nk_rawfb` prefixed functions alone and added `_ctx_` or `_img_` to the routines the appropriate get/set pixel functions that took in either a `struct rawfb_context` or `struct rawfb_image`. I also changed the to/from color function names because I have a thing against unreasonably long function names. Hopefully what I came up with is reasonable.

Reply:
Thanks for the feedback; should now all be addressed.